tests/suites/test_suite_ssl.function

```diff
@@ -77,6 +81,9 @@ typedef struct handshake_test_options
     void (*srv_log_fun)(void *, int, const char *, int, const char *);
     void (*cli_log_fun)(void *, int, const char *, int, const char *);
     int resize_buffers;
+#if defined(MBEDTLS_SSL_CACHE_C)
+    mbedtls_ssl_cache_context* cache;
```

Comment:
(minor) don't we do `mbedtls_ssl_cache_context *cache`?

Reply:
Fixed :)